gateway/src/shard/impl.rs

```diff
@@ -286,26 +414,35 @@ impl Shard {
     /// Send a command over the gateway.
     ///
     /// # Errors
-    /// Fails if command could not be serialized, or if the command could
-    /// not be sent.
     ///
-    /// Returns [`Error::Stopped`] if the shard isn't actively running.
+    /// Returns [`CommandError::Sending`] if the message could not be sent
+    /// over the websocket. This is indicative of the shard currently
```

Comment:
Nit: This sounds slightly strange to read out loud. What do you think about `This is indicative of the shard currently being restarted.`

Reply:
`This indicates the shard is currently restarting.`

Reply:
Went with the second suggestion since it's shorter and still gets the point across